cardigan/stories/components/ImageGallery.js

```diff
@@ -0,0 +1,22 @@
+import { storiesOf } from '@storybook/react';
+import ImageGallery from '../../../common/views/components/ImageGallery/ImageGallery';
+import { captionedImage } from '../content';
+import Readme from '../../../common/views/components/ImageGallery/README.md';
+import { boolean } from '@storybook/addon-knobs/react';
+
+const ImageGalleryExample = () => {
+  const images = [...new Array(5)].map(i => captionedImage());
+  return (
+    <ImageGallery
+      items={images}
+      id="cardy"
+      isStandalone={boolean('Is standalone?', false)}
+    />
+  );
+};
+
+const stories = storiesOf('Components', module);
+stories.add('ImageGallery', ImageGalleryExample, {
+  info: Readme,
+  isFullScreen: true,
```

Comment:
How and when do we decide about this? Isn't it an option in the interface?

Reply:
Our default decorator adds 30px of padding around each component in Storybook, to make it look nice. But when a component is full-bleed, removing this padding gives a more honest representation of the component.